src/frontend/screens/ObservationsList/ObservationListItem.tsx

```diff
@@ -33,20 +37,25 @@ function ObservationListItemNotMemoized({
   onPress = () => {},
 }: ObservationListItemProps) {
   const {preset} = useObservationWithPreset(observation.docId);
-  const deviceId = '';
+  const {data: deviceInfo} = useDeviceInfo();
+  const {deviceId} = deviceInfo || {};
 
   // const photos = !observationQuery.data ? [] : filterPhotosFromAttachments(
   //   observationQuery.data && observationQuery.data.attachments
   // ).slice(0, 3);
   const photos = [];
-  const isMine = observation.createdBy === deviceId;
+  const {data: createdByDeviceId} = useCreatedByToDeviceId(
+    observation.createdBy,
+  );
+  const isMine = createdByDeviceId === deviceId;
+
   return (
     <TouchableHighlight
       onPress={() => onPress(observation.docId)}
       testID={testID}
       style={{flex: 1, height: 80}}>
       <View
-        style={[styles.container, style, !isMine && styles.syncedObservation]}>
+        style={[styles.container, style, isMine && styles.syncedObservation]}>
```

Comment:
the `syncedObservation` style should be applied to observations from others, not your own

```suggestion
        style={[styles.container, style, !isMine && styles.syncedObservation]}>
```